src/hooks/useRelatedVenues.ts

```diff
@@ -80,9 +138,12 @@ interface UseConnectRelatedVenuesReturn {
   isCurrentVenueLoaded: boolean;
 }
 
-export const useConnectRelatedVenues: ReactHook<
-  UseConnectRelatedVenuesProps,
-  UseConnectRelatedVenuesReturn
+/**
+ * @deprecated use useRelatedVenues instead
+ */
+export const useLegacyConnectRelatedVenues: ReactHook<
+  UseLegacyConnectRelatedVenuesProps,
+  UseLegacyConnectRelatedVenuesReturn
 > = ({ venueId, withEvents = false }) => {
```

Comment:
[TODO] Refactor the codebase to make use of `useRelatedVenues` rather than this legacy `useLegacyConnectRelatedVenues`. Be sure to look for anything that is indirectly consuming the state via `useSelectors` as well. Once there is nothing left that is relying on this, then we can delete it.

**Note:** For the locations that are using the `withEvents` version of this, we will need to implement a new api function/hook/similar for fetching them in an efficient way. This may be end up getting handled in https://github.com/sparkletown/sparkle/pull/1367

Reply:
fixed in https://github.com/sparkletown/sparkle/commit/07a9e0e7a64dc56936c88853fd02eb79077e38a0 + https://github.com/sparkletown/sparkle/commit/e905c452a719589a6779047deccdf8c30aa93519 + https://github.com/sparkletown/sparkle/commit/b061f26feec4fe48b393ac38d67d814f528d6990 (and the supporting commits that lead up to those)